pom.xml

```diff
@@ -252,6 +252,7 @@
         <leveldb.version>1.8</leveldb.version>
         <lz4.version>1.3.0</lz4.version>
         <metrics.version>3.1.2</metrics.version>
+        <alt.metrics.version>3.2.6</alt.metrics.version>
```

Comment:
This `alt.metrics.version` addition here is (intentionally) an unmergeable hack. #6580 is to potentially upgrade the actual `metrics.version` itself.

Reply:
Please go ahead and try to update `metrics.version` to 3.2.6.